app/scripts/lib/sms-errors.js

```diff
@@ -0,0 +1,113 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * List of SMS error codes.
+ * From https://docs.nexmo.com/messaging/sms-api/api-reference#status-codes
+ */
+define((require, exports, module) => {
+  'use strict';
+
+  const _ = require('underscore');
+  const Errors = require('lib/errors');
+  const t = msg => msg;
+
+  const COULD_NOT_SEND = t('Could not send SMS');
+
+  /*eslint-disable sorting/sort-object-props*/
+  const ERRORS = {
+    SUCCESS: {
+      errno: 0,
+      message: t('SMS sent')
+    },
+    THROTTLED: {
+      errno: 1,
+      message: COULD_NOT_SEND
```

Comment:
We should use actual messages for these, even if we don't expect them to be user facing. The `message` shows in in Sentry and if they are all `Could not send SMS`, we won't be able to differentiate between errors.